bin/fuel-core/src/cli/run.rs

```diff
@@ -130,6 +125,20 @@ pub struct Command {
     )]
     pub database_type: DbType,
 
+    #[cfg(feature = "rocksdb")]
+    /// Define policy to apply to create columns
+    ///
+    /// If `Lazy`, columns will be created when an insertion happens
+    /// If `OnCreation`, columns will be created when opening DB.
+    /// When running a node in production we advise to use `OnCreation`
+    #[clap(
+        long = "rocksdb-columns-policy",
+        env,
+        value_enum,
+        default_value_t = ColumnsPolicy::default()
+    )]
+    pub rocks_db_columns_policy: ColumnsPolicy,
```

Comment:
I'm not sure how useful it is to make configurable=D Basically, `Lazy` exists for tests in most cases.

Reply:
I did it because I misunderstood this comment https://github.com/FuelLabs/fuel-core/pull/2526#discussion_r1900827467